python/cli/prelude_cli/templates/template.cs

```diff
@@ -5,7 +5,7 @@
 */
 using System;
 
-class TTP {
+class SecurityTest {
```

Comment:
Ok will need to update Build as well to keep these in sync

Reply:
PR is already opened

Reply:
https://github.com/preludeorg/build/pull/129

Reply:
Awesome!